ctapipe/tools/muon_reconstruction.py

```diff
@@ -120,4 +119,4 @@ def finish(self):
 
 def main():
     tool = MuonDisplayerTool()
-    tool.run()
+    return tool.run()
```

Comment:
I think this needs to be `sys.exit(tool.run())`, not `return` https://docs.python.org/2/library/sys.html#sys.exit